src/pages/components/data-table/usage.mdx

```diff
@@ -277,148 +280,312 @@ import {
+### When not to use
 
-</div>
+- When a more complex display of the data or interactions are required.
+- As a replacement for a spreadsheet application.
 
-<Caption>Basic data table</Caption>
+### Variations
 
-## Additional functionality
+| Type             | Purpose                                                                                                                                                                                                                             |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Basic_          | The basic data table comes with a base style with only the title, header, and table elements. rows.                                                                                                                                 |
+| _Zebra_          | The zebra data table features alternating colored rows to make scanning horizontal information easier for the user.                                                                                                                 |
+| _With selection_ | Batch actions are functions that may be performed on multiple items within a table. This type of table enables the user to select individual rows and apply an action. A batch action toolbar appears when table rows are selected. |
+| _With expansion_ | The expandable data table is useful for presenting large amounts of data in a small space. Rows are collapsed and can be expanded to reveal extra information.                                                                      |
 
-The following can be added to the data table to increase its functionality.
+## Formatting
 
-### Pagination
+### Anatomy
 
-Pagination divides table data into separate pages. Pagination is accompanied by an option that enables the user to change the number of items per page. See the [pagination](/components/pagination/usage) component for further guidelines.
+<Row>
+<Column colLg={12}>
 
-<div className="image--fixed">
+![data table anatomy](images/datatable-anatomy.png)
 
-![data table with pagination](images/data-table-usage-2.png)
+</Column>
+</Row>
 
-</div>
+1. **Title and description**: contains the data table title and an optional description.
+2. **Toolbar**: global data table controls including search and table settings.
+3. **Column header**: title for the row header with the optional sorting behavior.
+4. **Table row**: can be configured to show different types of data. Rows can be selectable, expandable, and modified to show alternating zebra stripe background colors.
+5. **Pagination table bar**: an optional component that lets a user navigate data as pages when the amount of data it too large to be shown at once.
 
-<Caption>Data table with various Pagination methods</Caption>
+### Sizing
```

Comment:
The the different sizes you mention in text should be in order of how you are showing it in the images.